### PR TITLE
docs: added interview airwallex blog

### PR DESCRIPTION
**Pull request by Serendipity96**
Changes:

docs: added interview airwallex blog

### CODE REVIEW

File: website/blog/2021-08-17-interview-airwallex.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Apache APISIX 在 Airwallex 的应用 | 专访 Airwallex 技术平台负责人李杨"
+keywords: 
+- APISIX
+- Airwallex
+- 空中云汇
+- 金融
+- 数据主权
+description: 当本文采访了 Airwallex 空中云汇 Technical Platform Lead 李杨，李杨负责公司技术平台的演进。采访中详细介绍了 Airwallex 空中云汇在做技术选型时为什么选择 Apache APISIX、Apache APISIX 在 Airwallex 空中云汇的使用场景、Apache APISIX 在生产环境的表现等。
```
Comment (juzhiyuan) [inline website/blog/2021-08-17-interview-airwallex.md]:
```suggestion
description: 本文采访了 Airwallex 空中云汇 Technical Platform Lead 李杨，李杨负责公司技术平台的演进。采访中详细介绍了 Airwallex 空中云汇在做技术选型时为什么选择 Apache APISIX、Apache APISIX 在 Airwallex 空中云汇的使用场景、Apache APISIX 在生产环境的表现等。
```

**Review by juzhiyuan — APPROVED** [review]

File: website/blog/2021-08-17-interview-airwallex.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+tags: [technology]
+---
+
+<!--truncate-->
+
+我们有幸采访到了 Airwallex 的技术平台负责人李杨，在采访中，李杨谈到了为什么在技术选型时选择 Apache APISIX ，以及 Apache APISIX 在 Airwallex 的应用。
+
+**Q：李博士，您好，请简单介绍一下自己和目前从事的工作。**
+
+**李杨：** 你好，我叫李杨（Jan Li），哲学博士，Apache APISIX Committer, Airwallex 空中云汇 Technical Platform Lead，负责公司技术平台的演进。在加入 Airwallex 之前，曾在万向区块链领导运链盟。在万向区块链之前，曾在花旗集团领导 OTC 衍生品风控平台。
```
Comment (yzeng25) [inline website/blog/2021-08-17-interview-airwallex.md]: Apache APISIX Committer,  ---> Apache APISIX Committer，

comma should change to Chinese

Comment (yzeng25) [reply website/blog/2021-08-17-interview-airwallex.md]: you might want to double check all symbols as well :;'",.()

File: website/blog/2021-08-17-interview-airwallex.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Airwallex （空中云汇）是一家全球金融科技公司，赋能各类规模的企业跨境运营，以此助力全球经济发展。以技术为核心，Airwallex（空中云汇）构建了专有的全球金融基础设施平台，全球支付网络已覆盖 130 多个国家和地区的 50 余种货币，为各类规模的企业提供数字化的金融科技产品，在全球互联的信息化时代，以更高效、安全的方式帮助企业在全世界高速发展。自 2015 年成立以来，Airwallex 空中云汇已获得来自顶级投资机构超 5 亿美元融资，目前在全球共有 12 个办公室及超过 900 名员工。
+
+![Airwallex LiYang](https://static.apiseven.com/202108/20210816001.png)
+
+**Q：在做技术选型的时候，是什么原因让您/您的技术团队选择使用 Apache APISIX ？**
+
+**李杨**：API 网关是极其重要的基础技术组件，在技术选型时我们主要从 6 个维度比较了主要的网关产品：
+
+- **稳定性**：API 网关的稳定性至关重要，在世界排名前 1000 的网站中，有 62.1% 是 Nginx 系的，这说明 Nginx 系 web server 是经过了生产环境复杂多样场景考验的；APISIX 全动态的设计也使得它能够在修改路由时不必 reload，client 的长链接也得以保持；我们也对 Apache APISIX 进行了压力测试，在 CPU 达到 70% 以上时 Apache APISIX 仍能稳定运转。
```
Comment (yzeng25) [inline website/blog/2021-08-17-interview-airwallex.md]: APISIX ---> Apache APISIX

**Review by yzeng25 — CHANGES_REQUESTED** [review]

File: website/blog/2021-08-17-interview-airwallex.md (hunk 2 above)
Comment (Serendipity96) [reply website/blog/2021-08-17-interview-airwallex.md]: thanks

**Review by yzeng25 — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Pull request merged**